src/components/combo-box/_combo-box.scss

```diff
@@ -11,7 +11,7 @@
 @import '../list-box/list-box';
 
 @include exports('combo-box') {
-  .bx--combo-box > .bx--list-box__field {
+  .#{$prefix}--combo-box > .#{$prefix}--list-box__field {
```

Comment:
Somewhat related to your PR, but I noticed that `ComboBox` has an extra `box-shadow` that could be bundled in with these changes 

<img width="240" alt="screen shot 2018-11-02 at 10 24 15 am" src="https://user-images.githubusercontent.com/11928039/47924331-7b944300-de89-11e8-98f8-5a4a41187186.png">

Reply:
Looks like there is box shadow and also the border-bottom on the input inside the list box? 🤔